### DISCUSSION TITLE
UX demo with Kindle display ED060XC3

### DISCUSSION THREAD

**Discussion by martinberlin** [vroland/epdiy#333] [Show and tell]
This is small demo using a Kindle paperwhite 1 display that you can find in Aliexpress for about 30 €

https://www.youtube.com/watch?v=5PBV6lDv1o4

Demonstrates the DU and GC16 draw modes.
PCB: https://github.com/martinberlin/epdiy-hardware-versions/tree/main/versions
If there is any interest I will make a pull request to add it to the main Hardware repository.

Reverse engineering of the TMA340 touch done with help of Dario [@ddB0515](https://x.com/ddB0515) (Note please make sure is that and not TMA445 since there are some slight differences)
Feedback is welcome!

![Screenshot from 2024-07-01 10-43-13](https://github.com/vroland/epdiy/assets/2692928/0a580009-dd59-4c12-9de8-69cc00b1b3d5)

**Comment by Zeron** [comment]
Any chance you could provide a link to the TMA430 code?

**Comment by martinberlin** [comment]
Sure.  You can find it here in the components submodule:

https://github.com/martinberlin/lv_port_esp32-epaper/tree/16/lvgl-90/components 
Please note that it will work only for TMA340 I never really tried with the other but there is some different initialization and someone tried and it doesn't work.
